contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -346,12 +346,18 @@ pub fn query_total_claimed(deps: Deps, stage: u8) -> StdResult<TotalClaimedRespo
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    let version = get_contract_version(deps.storage)?;
-    if version.contract != CONTRACT_NAME {
+    let version = CONTRACT_VERSION.to_string();
```

Comment:
You can use semver `Version` https://github.com/CosmWasm/cw-plus/blob/e2f07dbe133185f0c5ac195ae360fd10770e1b2f/contracts/cw20-ics20/src/contract.rs#L211